lib/api/controllers/routerController.js

```diff
@@ -54,7 +54,7 @@ module.exports = function RouterController (kuzzle) {
       executeFromRest.call(kuzzle, params, request, response);
     });
 
-    api.get('/_getStats', function (request, response) {
+    api.post('/_getStats', function (request, response) {
```

Comment:
With Kevin we talked about this yesterday, and we reached the conclusion that POST was the best verb to use... but we didn't think about changing the argument name nor the route name.

The problem with a ``getStatsSince`` route name is that it returns the last known frame if no argument is provided, and this is not coherent with what the route name suggests.  
So if we go for this solution, we might have to change the route behavior too, I think.

However, changing the argument name from ``timestamp`` to something like ``startTime`` would be easier to understand, so I think this is a good suggestion. This do not solve the GET vs POST debate, though.